src/topics/topicActions.js

```diff
@@ -40,25 +40,22 @@ export const fetchTopicsForStream = (narrow: Narrow) => async (
   dispatch(fetchTopics(stream.stream_id));
 };
 
-export const deleteMessagesForTopic = (streamName: string, topic: string) => async (
+export const deleteMessagesForTopic = (streamId: number, topic: string) => async (
   dispatch: Dispatch,
   getState: GetState,
 ) => {
   const state = getState();
   const outbox = getOutbox(state);
+  const currentStream = getStreamsById(state).get(streamId);
```

Comment:
Better to just call this `stream`. The extra word "current" isn't really adding any meaning here.

(Sometimes "current" is helpful when there's also a "previous" or "next", or some other notion of another thing of the same kind.)

... Oh I see, this is the name the existing code had.

But we're already touching all the places the old name was used. So the diff won't really get any bigger or more complex if we also fix up this name while we're here :slightly_smiling_face: